### DISCUSSION TITLE
VRCPlayer and AvatarPlayableController not found

### DISCUSSION THREAD

**Discussion by AndyK84** [200Tigersbloxed/UnityMods#14] [Q&A] [answered]
Hi, I wanted to tinker around a bit with the HrToVR Project since a friend of mine wanted to have the color change based on the HR value. But when I try to build the project I get a lot of VRCPlayer and AvatarPlayableController does not exist or could not be found messages. I imported all VRchat SDK and Unity dlls. What am I missing here?

**Comment by 200Tigersbloxed** [comment, marked as answer]
Hi, VRCPlayer and AvatarPlayableController should be in the Assembly-CSharp.dll

Also, you can use the HRPercent parameter to do what you're trying to do :)

https://github.com/200Tigersbloxed/UnityMods/issues/10

https://github.com/200Tigersbloxed/UnityMods/blob/main/HRtoVRChat/AvatarSetup.md#hrep

**Reply by AndyK84** [reply]
Hi, the Assembly-CSharp.dll is missing and I couldn't find it anywhere either. Where does it normally come from?

Oh the HRPercent parameter could be the perfect replacement. Thanks!

**Reply by AndyK84** [reply]
Nevermind, found it! Thanks a lot!
